spec/helpers/content_helper_spec.rb

```diff
@@ -10,4 +10,14 @@
       expect(helper.strip_tags(helper.preview(article))).to eq("ac")
     end
   end
+
+  describe '#html' do
+    it 'should call #link_glossary_terms if glossary option is given' do
+      expect(helper).to receive(:link_glossary_terms)
+      helper.html(article.body, glossary: true)
+
+      expect(helper).not_to receive(:link_glossary_terms)
```

Comment:
nit: describe the default behavior and the alternate behavior in separate expectations:
```
describe '#html' do
  let(:create_html) {  helper.html(article.body) }

  it 'does not call #link_glossary_terms' do
    expect(helper).not_to receive(:link_glossary_terms)
    create_html
  end

  context 'when glossary is true' do
    let(:create_html) { helper.html(article.body, glossary: true) }

    it 'calls #link_glossary_terms' do
      expect(helper).to receive(:link_glossary_terms)
      create_html
    end
  end
end
```